src/API/api.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+export const baseUrl = 'http://localhost:3001/';
+
+// custom is to alternate the endpoint
+export async function sendGet(custom = '') {
+  return axios.get(baseUrl + custom).then((response) => response.data);
+}
+
+// Custom route must be provided with ID for destroy to find the room
+export async function sendDelete(custom) {
+  return axios.delete(baseUrl + custom).then((response) => response.status);
+}
+
+export async function sendPost(custom, data) {
+  return axios
+    .post(baseUrl + custom, {
+      data,
+    })
+    .then((response) => response);
+}
```

Comment:
I think we should revise this function because is not posting data as it should be
![image](https://user-images.githubusercontent.com/99031215/191337967-31856f38-81dc-4f4d-bcf8-94c9b87400c7.png)

Reply:
@AxelSoler, I think it's related to how `json-server` works. when we send an actual API request to our own end point. this should work fine.